newIDE/app/src/MarketingPlans/MarketingPlanFeatures.js

```diff
@@ -0,0 +1,208 @@
+  gameFeaturings: ?(GameFeaturing[]),
+  requirementsErrors: React.Node[],
+  onPurchase: I18nType => Promise<void>,
+  isPlanActive: boolean,
+  hideBorder?: boolean,
+|};
+
+const MarketingPlanFeatures = ({
+  marketingPlan,
+  gameFeaturings,
+  requirementsErrors,
+  onPurchase,
+  isPlanActive,
+  hideBorder,
+}: Props) => {
+  const { limits } = React.useContext(AuthenticatedUserContext);
+  const gdevelopTheme = React.useContext(GDevelopThemeContext);
+  const {
+    id,
+    nameByLocale,
+    canExtend,
+    descriptionByLocale,
+    bulletPointsByLocale,
+  } = marketingPlan;
+
+  const activeGameFeaturings: ?(GameFeaturing[]) = React.useMemo(
+    () => {
+      if (!gameFeaturings) return null;
+
+      return gameFeaturings.filter(
+        gameFeaturing => gameFeaturing.expiresAt > Date.now() / 1000
+      );
+    },
+    [gameFeaturings]
+  );
+
+  const planCreditsAmount = getMarketingPlanPrice(marketingPlan, limits);
+  if (!planCreditsAmount) {
+    console.error(`Could not find price for marketing plan ${id}, hiding it.`);
+    return null;
+  }
+  const hasErrors = requirementsErrors.length > 0;
+
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <div
+          style={{
+            ...styles.campaign,
+            border: hideBorder
+              ? undefined
+              : isPlanActive
+              ? `2px solid ${gdevelopTheme.message.valid}`
+              : `1px solid ${gdevelopTheme.palette.secondary}`,
+          }}
+          key={id}
+        >
+          <ColumnStackLayout
+            alignItems="center"
+            justifyContent="space-between"
+            noMargin
+            expand
+          >
+            <div style={styles.titleContainer}>
+              <LineStackLayout
+                justifyContent="space-between"
+                alignItems="flex-start"
+                expand
+                noMargin
+              >
+                <LineStackLayout noMargin alignItems="flex-start">
+                  {getIconForMarketingPlan(marketingPlan)}
+                  <Text size="sub-title">
+                    {selectMessageByLocale(i18n, nameByLocale)}
+                  </Text>
+                </LineStackLayout>
+                <Text size="body-small" color="secondary">
+                  <Trans>{planCreditsAmount} credits</Trans>
+                </Text>
+              </LineStackLayout>
+            </div>
+
+            <div style={styles.bulletPointsContainer}>
+              {hasErrors
+                ? requirementsErrors.map((error, index) => (
+                    <Column key={index} expand noMargin>
+                      <Line noMargin alignItems="center">
+                        <CheckCircle
+                          style={{
+                            ...styles.bulletIcon,
+                            color: gdevelopTheme.message.error,
+                          }}
+                        />
+                        <Text style={{ flex: 1 }}>{error}</Text>
```

Comment:
maybe extract that text style (you're using it below too)

Reply:
or put it in a `Line expand` ?

Reply:
I did not change that, it was there before, so I'm not sure using a Line would have the same effect. I'll put the style in the `styles` object